plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/AccumuloQueryRunner.java

```diff
@@ -128,8 +128,7 @@ private static void copyTable(
                 break;
         }
 
-        LOG.info("Running import for %s", target, sql);
-        LOG.info("%s", sql);
+        LOG.info("Running import for %s%n%s", target, sql);
```

Comment:
```suggestion
        LOG.info("Running import for %s", target);
        LOG.info("Query %s", sql);
```

Maybe

Reply:
That would be a more straightforward transformation, sure, but I think that if they are two separate logs then they may get separated in the output if there are concurrent threads. Perhaps this merging can be done in a separate commit, though.